added hint where to find the NugetApiKey for users reading the online doc

jantari opened the pull request:
1. Standardize on jinja2-style for REPLACE_ME-type values
2. The NugetApiKey variable was called NexusApiKey in one place (fixed, consistency)
3. Added the information where to find the Nuget/Nexus API key as I didn't know when following the online readme

Number 1 and 3 are things that irritated me personally when setting up QDE so I fixed them.
Number 2 is just something I noticed when preparing the PR just now.

I think the docs should standardize on one convention of values that the user must replace inside of commands or files etc. - since the Jinja2-style is used more often currently and because it is more compact I chose that one. When I first ran the `Set-QDEnvironment.ps1` script in my QDE i replaced the Thumbprint and DNS-Name with my own valued but not the ApiKey as I thought maybe the script itself expects that jinja2-style placeholder. Making the values the user should replace use one stlye-convention removes this possible mistake.

vexx32 commented on input/en-us/quick-deployment/setup/desktop-readme.md:
```diff
@@ -100,18 +100,18 @@ You will need either the certificate thumbprint or the subject in order to use t
 # If you have the thumbprint, run:
-& C:\choco-setup\files\Set-QDEnvironment.ps1 -NexusApiKey "{{NugetApiKey}}" -CertificateThumbprint "INSERT_CERTIFICATE_THUMBPRINT_HERE" -CertificateDnsName "INSERT_DNS_NAME_HERE"
+& C:\choco-setup\files\Set-QDEnvironment.ps1 -NexusApiKey "{{NugetApiKey}}" -CertificateThumbprint "{{CertificateThumbprint}}" -CertificateDnsName "{{DnsName}}"
 
 # If you have the subject instead, run:
-& C:\choco-setup\files\Set-QDEnvironment.ps1 -NexusApiKey "{{NugetApiKey}}" -CertificateSubject "INSERT_CERTIFICATE_SUBJECT_HERE" -CertificateDnsName "INSERT_DNS_NAME_HERE"
+& C:\choco-setup\files\Set-QDEnvironment.ps1 -NexusApiKey "{{NugetApiKey}}" -CertificateSubject "{{CertificateThumbprint}}" -CertificateDnsName "{{DnsName}}"
```
These are intended to be replaced by the user specifically when they go to run the script. The rest of the `{{replacement}}` values in here are replaced automatically when the QDE instance is setup and aren't intended to be user-replaceable, which is why there's a difference in style. Some folks take things a bit literally when running commands they're given, so showing it without additional braces etc tends to minimize accidental mishaps.

jantari replied:
Okay so I understand correctly that the `{{ jina2 style }}` variables are automatically replaced e.g. when the QDE generates the Readme.html for the desktop? That would make sense then and I'll revert this. Thanks

vexx32 replied:
There is automatic substitution yeah, not happening here of course for this version of the doc, but we kept it the same to minimize conflicts when we need to update it so we don't get too many differences between the versions, it gets a bit fun trying to avoid errors. The `{{NugetApiKey}}` is one of those as well, eyah.

vexx32 commented on input/en-us/quick-deployment/setup/desktop-readme.md:
````diff
@@ -262,7 +262,7 @@ This will be encrypted. To setup, do the following:
 
 ```powershell
-choco apikey add --key="'{{NexusApiKey}}'" --source="'https://chocoserver:8443/repository/ChocolateyInternal/'"
+choco apikey add --key="'{{NugetApiKey}}'" --source="'https://chocoserver:8443/repository/ChocolateyInternal/'"
````
Huh, guess we missed renaming this one, thanks!

vexx32 reviewed (commented):
We don't necessarily want to change one part of this, but the rest looks good. Thanks! 💖

steviecoaster commented on input/en-us/quick-deployment/setup/desktop-readme.md:
````diff
@@ -100,18 +100,18 @@ You will need either the certificate thumbprint or the subject in order to use t
 ```powershell
 # If you have the thumbprint, run:
-& C:\choco-setup\files\Set-QDEnvironment.ps1 -NexusApiKey "{{NugetApiKey}}" -CertificateThumbprint "INSERT_CERTIFICATE_THUMBPRINT_HERE" -CertificateDnsName "INSERT_DNS_NAME_HERE"
+& C:\choco-setup\files\Set-QDEnvironment.ps1 -NexusApiKey "{{NugetApiKey}}" -CertificateThumbprint "{{CertificateThumbprint}}" -CertificateDnsName "{{DnsName}}"
````
Please revert this change. This will just cause support tickets, as people will undoubtedly attempt to pass the brackets with their commands

steviecoaster commented on input/en-us/quick-deployment/setup/desktop-readme.md:
```diff
@@ -100,18 +100,18 @@ You will need either the certificate thumbprint or the subject in order to use t
 
 # If you have the subject instead, run:
-& C:\choco-setup\files\Set-QDEnvironment.ps1 -NexusApiKey "{{NugetApiKey}}" -CertificateSubject "INSERT_CERTIFICATE_SUBJECT_HERE" -CertificateDnsName "INSERT_DNS_NAME_HERE"
+& C:\choco-setup\files\Set-QDEnvironment.ps1 -NexusApiKey "{{NugetApiKey}}" -CertificateSubject "{{CertificateThumbprint}}" -CertificateDnsName "{{DnsName}}"
```
This will need reverted as well. For consistency I think replacing all of the `{{}}` with just straight quotes and a descriptive placeholder is better served.

pauby commented:
I am in agreement with [Stevie here](https://github.com/chocolatey/docs/pull/95/files#r542431213). Jinga templating is great for techies who understand what it is, but there are a lot of techies, and non-techies, who don't know what it is. We need to cater to everybody.

jantari commented:
Ok thanks for the feedback - I reverted that change and thus also renamed the PR

Also goes to show I still need to force myself to make more atomic commits to allow cherry-picking this kind of stuff *argh*

vexx32 commented on input/en-us/quick-deployment/setup/desktop-readme.md:
```diff
@@ -103,15 +103,15 @@ You will need either the certificate thumbprint or the subject in order to use t
 
 # If you have the subject instead, run:
-& C:\choco-setup\files\Set-QDEnvironment.ps1 -NexusApiKey "{{NugetApiKey}}" -CertificateSubject "INSERT_CERTIFICATE_SUBJECT_HERE" -CertificateDnsName "INSERT_DNS_NAME_HERE"
+& C:\choco-setup\files\Set-QDEnvironment.ps1 -NexusApiKey "{{NugetApiKey}}" -CertificateSubject "INSERT_CERTIFICATE_THUMBPRINT_HERE" -CertificateDnsName "INSERT_DNS_NAME_HERE"
```
Double check this one, I think possibly just a copy paste error? 🙂

jantari replied:
You're right, thanks - fixed!

vexx32 reviewed (changes requested):
One last bit, probably just an accidental error. Otherwise this looks good, thanks!

vexx32 reviewed (approved):
Looks good, thanks for this! 💖 😊

gep13 reviewed (approved):
LGTM!

gep13 commented:
@jantari your changes have been merged, thanks for your contribution 👍
